srcpkgs/base-files/files/profile

```diff
@@ -2,8 +2,23 @@
 
 # System wide environment and startup programs.
 
+appendpath () {
+    case ":$PATH:" in
+        *:"$1":*)
+            ;;
+        *)
+            PATH="${PATH:+$PATH:}$1"
+    esac
+}
+
 # Set our default path (/usr/sbin:/sbin:/bin included for non-Void chroots)
-PATH="/usr/local/sbin:/usr/local/bin:/usr/bin:/usr/sbin:/sbin:/bin"
+appendpath /usr/local/sbin
+appendpath /usr/local/bin
+appendpath /usr/bin
+appendpath /usr/sbin
+appendpath /sbin
+appendpath /bin
```

Comment:
@maxice8, I kept the order as in the master branch, but shouldn't `/usr/sbin` goes before `/usr/bin` as in other distros?